students/cindywaldron/lesson2/drawGrid.py

```diff
@@ -0,0 +1,21 @@
+def printGrid(x,y):
```

Comment:
per pep8 style guidelines function names should be snake_case style -> `print_grid`